ibft2-eks/cfn-besu-ibft2-eks.yml

```diff
@@ -0,0 +1,340 @@
+---
+AWSTemplateFormatVersion: '2010-09-09'
+Description: "Besu IBFT2 EKS Template"
+
+Parameters:
+
+  VpcId:
+    Default: "vpc-"
+    Type: "AWS::EC2::VPC::Id"
+    Description: The ID of your existing VPC that we will deploy into (e.g., vpc-012345)
+
+  PublicSubnetAId:
+    Default: "subnet-"
+    Description: The ID of the public subnet in AZ 1
+    Type: String
+
+  PublicSubnetBId:
+    Default: "subnet-05990c77cc8bf62a8"
```

Comment:
I think this default needs to be changed to be a fake subnet ID

Reply:
done